<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPP_ProjectFinalClass</class>
 <widget class="QWidget" name="CPP_ProjectFinalClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CPP_ProjectFinal</string>
  </property>
  <widget class="QWidget" name="BoardPage" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1081</width>
     <height>601</height>
    </rect>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>581</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tw Cen MT Condensed</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>INPUT</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tw Cen MT Condensed</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>OUTPUT</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tw Cen MT Condensed</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>GATE</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tw Cen MT Condensed</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ADDER SUBTRACTOR</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tw Cen MT Condensed</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CODE CONVERTER</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lblDescT">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>520</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblDesc">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>539</y>
      <width>881</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="titlePage" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>481</width>
     <height>301</height>
    </rect>
   </property>
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffaa39; border:none;</string>
    </property>
    <property name="text">
     <string>START CONNECTING</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="bgImg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CPP_ProjectFinal.qrc"/>
 </resources>
 <connections/>
</ui>
